src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -18,3 +18,9 @@
  ::wallet-add-custom-token
  (fn [{:keys [db]}]
    {:db (dissoc db :wallet/custom-token-screen)}))
+
+(fx/defn hide-buy-crypto
+  {:events [::hide-buy-crypto]}
+  [{:keys [db]}]
+  ;; TODO: Think where to persist this preference
```

Comment:
@cammellos what do you think, where it is better to persist this?

Reply:
not sure, I would probably store it in `asyncStorage` as it seems very mobile centric, we could move it to status-go eventually. 
In terms of UX as a user i would expect that once I dismiss it, I won't see it anymore, even after restarting the app.
What do you think?

Reply:
Added `asyncStorage`